### PR TITLE
Implemented calling stepping speed

### PR DESCRIPTION
**Pull request by MatAdam333**
Vanú jfunkc ió

### CODE REVIEW

File: train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java [hunk 1]
```diff
@@ -2,22 +2,37 @@
 
 import hu.bme.mit.train.interfaces.TrainController;
 
+import java.util.Timer;
+import java.util.Timer.TimerTask;
```
Comment (SolyomBalint) [inline train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java:6]: Is this good?

File: train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java [hunk 2]
```diff
@@ -2,22 +2,37 @@
...
 
 	private int step = 0;
 	private int referenceSpeed = 0;
 	private int speedLimit = 0;
+	private Timer timer = new Timer();
+
+	public TrainControllerImpl() {
+
+		java.util.TimerTask tt = new TimerTask(){
+			@Override
```
Comment (SolyomBalint) [inline train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java:18]: This is not a good naming.

**Review by SolyomBalint — CHANGES_REQUESTED** [review]

File: train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java (hunk 1 above)
Comment (MatAdam333) [reply train-controller/src/main/java/hu/bme/mit/train/controller/TrainControllerImpl.java:6]: No :_(

**Comment by SolyomBalint** [comment]
Mine is better

**Pull request closed without merging**